### DISCUSSION TITLE
lotus sync wait -- command exec sync complete but  can not down!

### DISCUSSION THREAD

**Discussion by peterz3g** [filecoin-project/lotus#5982] [Build and Sync] [answered]
I start the dev net at local.  machine1 is the genesis node, from machine2, I connect to machine1 success and exec the sync command line to sync the new state.  

machine1 and machine2 is run in docker.

but the command waiting at below state.  todo is 0 , but command can not finished , I am waiting for 10 hours more now.  

how can i fix this problem. thanks!  

==============================================

root@:~/lotus_dev/lotus# ./lotus -v
lotus version 1.7.0-dev+2k+git.f10bb9e23

root@:~/lotus_dev/lotus# ./lotus sync wait
Worker: 48; Base: 290; Target: 291 (diff: 1)
State: complete; Current Epoch: 291; Todo: 0
Validated 2 messages (0 per second)

================wait here, not down, not out, not over=================================

**Comment by de-authority** [comment]
I have same problems ~

**Reply by hcm609** [reply]
Did you guys solved it somehow?

**Comment by SgtCoin** [comment, marked as answer]
Archiving for future reference due to age.
